Rethinking/end-of-chapter-practice-problems/ch-2.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>Typo: The vet administers</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Thanks! Fixed it.